### PR TITLE
Fix pom and general structure

### PR DESCRIPTION
**Pull request by Maxopoly**

### CODE REVIEW

**Comment by Maxopoly** [comment]
Makes 2 separate jars now (one for bukkit and one for bungee), but all the overhead is gone

**Comment by ProgrammerDan** [comment]
so the only build issue I encountered is that bungeecord-proxy doesn't appear to have a public repository; assuming that that is a real requirement perhaps just make a note in the README that building BungeeCord locally is required before building this.

Other then that no objections to the merge, looks great

File: BetterShardsBungee/src/main/java/vg/civcraft/mc/bettershards/bungee/BetterShardsBungee.java [hunk 1]
```diff
@@ -62,11 +62,12 @@ private void loadDB() {
 	public void loadConfiguration() {
 		if (!getDataFolder().exists())
 			getDataFolder().mkdir();
+		
 		configManager = ConfigurationProvider.getProvider(YamlConfiguration.class);
 		if (!configFile.exists()) {
 			try {
                 configFile.createNewFile();
-                try (InputStream is = getResourceAsStream("bungee" + File.separator + "config.yml");
+                try (InputStream is = getResourceAsStream("config.yml");
```
Comment (rourke750) [inline BetterShardsBungee/src/main/java/vg/civcraft/mc/bettershards/bungee/BetterShardsBungee.java:70]: Why did you change this? Won't there be duplicate config from bukkit and bungee.

File: BetterShards/pom.xml [hunk 2]
```diff
@@ -1,101 +0,0 @@
...
-				<targetPath>.</targetPath>
-				<filtering>true</filtering>
-				<directory>../BetterShardsBungee</directory>
-				<includes>
-					<include>bungee.yml</include>
-				</includes>
-			</resource>
-			<!-- config.yml needs its own location so it does not mess with the bukkit plugin. -->
-			<resource>
-				<targetPath>./bungee</targetPath>
```
Comment (rourke750) [inline BetterShards/pom.xml:53]: Having this was useful since both bukkit and bungee config are called config.yml. When they are shaded they will both be the same. This should be kept since we are going to be moving away from the civmodcore and including straight config.yml files in the future.

Comment (ProgrammerDan) [inline BetterShards/pom.xml:53]: @Rourke750 For a better way to do it, look at https://github.com/ProgrammerDan/SkilUp

I didn't look close enough here to see if Max is already doing it.

But basically in the SkilUp-light refactor I have separate config.yml for bungee and bukkit, there is no conflict and no special jar resource handling necessary as bukkit doesn't shade in bungee or vice versa. They both shade in common class files but no common resources.

File: BetterShardsBungee/src/main/java/vg/civcraft/mc/bettershards/bungee/BetterShardsBungee.java (hunk 1 above)
Comment (Maxopoly) [inline BetterShardsBungee/src/main/java/vg/civcraft/mc/bettershards/bungee/BetterShardsBungee.java:70]: No, it's separate jars now

Comment (rourke750) [inline BetterShardsBungee/src/main/java/vg/civcraft/mc/bettershards/bungee/BetterShardsBungee.java:70]: Oh perfect.

**Comment by ProgrammerDan** [comment]
Any other issues @rourke750 or is this good to merge?

**Comment by rourke750** [comment]
Looks good.

**Comment by ProgrammerDan** [comment]
So the one thing we missed all these months ago is that it immediately broke everything that expects the combined jar, and we never bothered to fix / replace it.

Woops...
